src/components/PersonalData/Certyfications/Certyfications.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const certyfications = () => {
```

Comment:
I don't know how big it will be but for now I don't see the point in separated component for that